From 1cc8524946d5b8985dbf420126b12a3293811716 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:07:17 -0400
Subject: [usb] Revert: serial: add usbid for dell wwan card to sierra.c

Message-id: <1410887463-20674-366-git-send-email-dzickus@redhat.com>
Patchwork-id: 95329
O-Subject: [RHEL7 PATCH 365/591] Revert "USB: serial: add usbid for dell wwan card to sierra.c"
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 2e01280d2801c72878cf3a7119eac30077b463d5
Author: Johan Hovold <jhovold@gmail.com>
Date:   Fri Mar 28 18:05:10 2014 +0100

    Revert "USB: serial: add usbid for dell wwan card to sierra.c"

    This reverts commit 1ebca9dad5abe8b2ed4dbd186cd657fb47c1f321.

    This device was erroneously added to the sierra driver even though it's
    not a Sierra device and was already handled by the option driver.

    Cc: Richard Farina <sidhayn@gmail.com>
    Cc: stable@vger.kernel.org
    Signed-off-by: Johan Hovold <jhovold@gmail.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/serial/sierra.c b/drivers/usb/serial/sierra.c
index a9eb622..6b192e6 100644
--- a/drivers/usb/serial/sierra.c
+++ b/drivers/usb/serial/sierra.c
@@ -291,7 +291,6 @@ static const struct usb_device_id id_table[] = {
  { USB_DEVICE(0x0f3d, 0x68A3),  /* Airprime/Sierra Wireless Direct IP modems */
    .driver_info = (kernel_ulong_t)&direct_ip_interface_blacklist
  },
-       { USB_DEVICE(0x413C, 0x08133) }, /* Dell Computer Corp. Wireless 5720 VZW Mobile Broadband (EVDO Rev-A) Minicard GPS Port */
 
  { }
 };
-- 
1.7.1